### ISSUE TITLE
Crystallization in non-covid images

### ISSUE THREAD

**Issue by GMBarra** [UCSD-AI4H/COVID-CT#37]
Hello, thank you very much for making this dataset available. I was looking at the non-covid images and I realized that even though they are classified as non-covid, a "crystallization" similar to covid is observed, do you know what other type of disease it is, if it is not covid?

Non-Covid: 

![image](https://user-images.githubusercontent.com/25462755/99839650-d7ea5e00-2b49-11eb-9e5b-5748de05801e.png)

Covid: 

![image](https://user-images.githubusercontent.com/25462755/99839667-de78d580-2b49-11eb-99c6-c39b1fa7b21c.png)

**Comment by Xarlley** [comment]
I agree with you. Actually, when I use my machine-learning model to analysis the non-convid CT images provided here, my model comes to conclusion that most of these non-convid CT images should be convid.
